artifactory_cleanup/cli.py

```diff
@@ -83,6 +98,46 @@ def _get_today(self):
             print(f"Simulating cleanup actions that will occur on {today}")
         return today
 
+    def _print_table(self, result: dict):
+        table = PrettyTable()
+        table.field_names = ["Cleanup Policy", "Files count", "Size"]
+        table.align["Cleanup Policy"] = "l"
+
+        for policy_result in result["policies"]:
+            row = [
+                policy_result["name"],
+                policy_result["file_count"],
+                size(policy_result["size"]),
+            ]
+            table.add_row(row)
+
+        table.add_row(["", "", ""])
+        table.add_row(["Total size: {}".format(size(result["total_size"])), "", ""])
+        print(table)
+
+    def _create_output_file(self, result, filename, format):
+        with open(filename, "w") as file:
+            if format == "table":
+                table = PrettyTable()
```

Comment:
Let's add `_format_table(result)` function - I see we repeat table generation code twice